Shaders/RHI/BindlessPrototype.azsl

```diff
@@ -131,21 +132,28 @@ PixelOutput MainPS(VertexOutput psInput)
         BindlessMaterial1 bindlessMaterial1; 
         ReadFromFloatBuffer(bindlessMaterial1.m_diffuseColor, HandleSrg::m_materialHandle, offset);
         ReadFromFloatBuffer(bindlessMaterial1.m_diffuseTextureIndex, HandleSrg::m_materialHandle, offset);
-
-        Texture2D texture = ImageSrg::m_textureArray[bindlessMaterial1.m_diffuseTextureIndex % 8]; // % 8 for wrap-around texture index as specified in ImageSrg.m_textureArray
-        OUT.m_color = texture.Sample(ImageSrg::m_sampler, psInput.m_uv);
+        
+#ifdef UB_DIRECTBINDING_NOTSUPPPORTED 
```

Comment:
Is this not enabled for metal? I can't seem to find it.

Reply:
It's defined here - https://github.com/aws-lumberyard-dev/o3de/blob/rgba16f/BindlessUpdateNov2022/Gems/Atom/Asset/Shader/Code/AZSL/Platform/Mac/Metal/AzslcHeader.azsli